### ISSUE TITLE
Does Eclim support the following features for Python development?

### ISSUE THREAD

**Issue by alphaCTzo7G** [ervandew/eclim#572]
Hi all,

Currently, I use Vim 8 with YouCompleteMe, Syntastic, and Python-mode, pretty
much work in tmux/zsh. I like the workflow, but I also understand that
IDE's sometimes provide benefits when working on large codebases. Towards that, I
am considering using Eclim/Pydev with VIM running in Tmux on Ubuntu 16.04. I pretty much work only in Python these days. 

Things I am looking for Python development
1. better code completion compared to YouCompleteMe
2. Code Navigation (GoToDefinition)
3. Code browsing capabilities
4. Refactoring

If I run Pydev in the background, and communicate with it through VIM, will I
be able to get these functionalities?

**Comment by alnavv** [comment]
yes, you can get almost everything you asked for.  Refactoring is still a hard job (though rope is useful).
if you don't mind to use neovim instead of vim (and you should), you can check the scripts i am preparing, you might get  a working system easily: https://github.com/alnavv/nvim-IDE (neovim+youcompleteme+eclim+eclipse). Feel free to ask if you need something else.

**Comment by alphaCTzo7G** [comment]
>Code browsing capabilities
>Little too vague for me to really comment on.

I meant.. something like taglist or tagbar with gutentags/ctags?

Is that possible?

**Comment by alphaCTzo7G** [comment]
@alnavv.. thanks.. that almost what I am looking for.. I will try it out on a fresh VM

Should this work on any generic linux distro, or do you have this working on Mac?
